src/main/java/de/mediathekview/mlib/daten/ListeFilme.java

```diff
@@ -17,21 +17,32 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-package mSearch.daten;
-
-import mSearch.Const;
-import mSearch.tool.Duration;
-import mSearch.tool.FileSize;
-import mSearch.tool.Functions;
-import mSearch.tool.Log;
-import org.apache.commons.lang3.time.FastDateFormat;
+package de.mediathekview.mlib.daten;
 
 import java.security.MessageDigest;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.LinkedList;
+import java.util.Optional;
+import java.util.SimpleTimeZone;
+import java.util.TreeSet;
 import java.util.concurrent.ConcurrentSkipListSet;
 
+import org.apache.commons.lang3.time.FastDateFormat;
+
+import de.mediathekview.mlib.Const;
+import de.mediathekview.mlib.tool.Duration;
+import de.mediathekview.mlib.tool.FileSize;
+import de.mediathekview.mlib.tool.Functions;
+import de.mediathekview.mlib.tool.Log;
+
 @SuppressWarnings("serial")
 public class ListeFilme extends ArrayList<DatenFilm> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)